Source/WebCore/platform/graphics/avfoundation/objc/SourceBufferPrivateAVFObjC.mm

```diff
@@ -934,10 +931,6 @@ static inline bool shouldAddContentKeyRecipients()
 AVSampleBufferAudioRenderer *SourceBufferPrivateAVFObjC::audioRendererForTrackID(TrackID trackID) const
 ALLOW_NEW_API_WITHOUT_GUARDS_END
 {
-    if (!m_audioRenderers.size())
-        return nil;
-    if (m_audioRenderers.size() == 1)
-        return m_audioRenderers.begin()->second.get();
     if (auto it = m_audioRenderers.find(trackID); it != m_audioRenderers.end())
         return it->second.get();
     return nil;
```

Comment:
I would tend to write it as:
```
auto it = m_audioRenderers.find(trackID);
return it != m_audioRenderers.end() : it->second.get() : nil;
```
Also why not returning a RetainPtr?

Reply:
done